tools/docker/start.sh

```diff
@@ -17,13 +17,15 @@ REF=${2:-master}
 
 cd ~
 
-mkdir web-platform-tests
-cd web-platform-tests
+if [ ! -d web-platform-tests ]; then
```

Comment:
Maybe I'm being stupid, but is this actually right? Will the Docker image be recreated when REF changes?

Reply:
This runs in the docker process and affects the overlay filesystem which has a lifetime which is that of the specific instance, not of the underlying image. So yes, I think this is correct.